sdk/purview/purview-catalog-rest/test/public/glossary.spec.ts

```diff
@@ -63,13 +67,13 @@ describe("purview catalog glossary test", () => {
       });
 
     console.log("LRO init resp: ", initialResponse);
-    const poller = getLongRunningPoller(client, initialResponse, {
+    const poller = await getLongRunningPoller(client, initialResponse, {
       intervalInMs: 100,
     });
 
     const result = await poller.pollUntilDone();
     console.log("LRO polling result:", result);
-    if (result.status === "500") {
+    if (isUnexpected(result)) {
```

Comment:
ci was failed at this, in recording, the final status was 200 which was not matched with the sdk behaviour:
https://github.com/kazrael2119/azure-sdk-for-js/blob/ba0137371153b9a417a398ad1f6285574a4c2699/sdk/purview/purview-catalog-rest/src/clientDefinitions.ts#L832-L841

here is the swagger def:
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2022-03-01-preview/purviewcatalog.json#L2841-L2893